tests/test-sources/plugins/lsp/conform-nvim.nix

```
@@ -59,4 +59,39 @@
       };
     };
   };
+
+  custom_format_on_save_function = {
+    plugins.conform-nvim = {
+      enable = true;
+
+      formattersByFt = {
+        lua = ["stylua"];
+        python = ["isort" "black"];
+        javascript = [["prettierd" "prettier"]];
```

```suggestion
        javascript = ["prettierd" "prettier"];
```
Isn't this supposed to be "single bracketed" ?

---

Yeah wonder if there is something from Arch or one of my configs somehow leaking into the Nix build env....

Thanks will remove those extra brackets now.

---

> Isn't this supposed to be "single bracketed" ?

Actually no just remembered this is the like this for a reason:

https://github.com/stevearc/conform.nvim?tab=readme-ov-file#setup

---

Oh fine !